exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/otlp/logs/LogMarshaler.java

```diff
@@ -36,6 +38,7 @@ final class LogMarshaler extends MarshalerWithSize {
   private final TraceFlags traceFlags;
   @Nullable private final String traceId;
   @Nullable private final String spanId;
+  private final byte[] eventName;
```

Comment:
It's a little inconsistent to me here -- I grok that not every log has a span/trace context, so it makes sense that the `traceId` and `spanId` are `@Nullable`. But isn't the same true of `eventName`? Not every log has an event name, so why not also make it `@Nullable` instead of doing the empty array thing?

Perhaps there's a broader practice regarding protos and arrays and I'm just not familiar with that? Or more likely it's just that we always favor non-null fields and this gets marshaled to length 0 anyway so it doesn't matter....in which case, I also wonder why we didn't do that with empty non-null strings for span/trace ID (or use the invalid constants). Anyway, not a big deal, just made me wonder.

Reply:
I just followed the pattern set a few lines above by `private final byte[] severityText` 🤷‍♂️